CHANGELOG.md

```diff
@@ -88,7 +88,6 @@ Version number updated to 3.4 to make numbering consistent with Mattermost serve
 ### Known Issues
 
 #### Windows
- - Copying a link address and pasting it inside the app doesn't work
```

Comment:
Your obviously right. When I drafted this, I hoped it would still make it to release ;)
Will change.